.github/workflows/staging-smoke-tests.yaml

```diff
@@ -1,17 +1,67 @@
 name: DataRepo Staging Smoke Tests
 env:
-  K8_CLUSTER: 'staging-master'
-  # This must be defined for the bash redirection
-  GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
-  # This must be defined for the bash redirection
-  GOOGLE_SA_CERT: 'jade-dev-account.pem'
+  K8_CLUSTER: jade-master-us-central1
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/staging-test-runner.json
+  GOOGLE_CLOUD_PROJECT: terra-datarepo-staging
+  GOOGLE_CLOUD_DATA_PROJECT: terra-datarepo-staging-data
 on:
   workflow_dispatch: {}
 jobs:
   test-runner-staging:
     runs-on: ubuntu-latest
     steps:
-      - name: "Checkout jade-data-repo develop branch"
+      - name: "Checkout jade-data-repo"
         uses: actions/checkout@v2
+      - name: "Import Vault staging secrets"
+        uses: hashicorp/vault-action@v2.0.1
         with:
-          ref: develop
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.STAGING_ROLE_ID }}
+          secretId: ${{ secrets.STAGING_SECRET_ID }}
+          secrets: |
+            secret/dsde/datarepo/staging/test-runner-sa key | B64_APPLICATION_CREDENTIALS ;
+      - name: "Perform IAM policy cleanup for staging"
+        run: |
+          # write vault tokens
+          base64 --decode <<< ${B64_APPLICATION_CREDENTIALS} > ${GOOGLE_APPLICATION_CREDENTIALS}
+
+          gcloud auth activate-service-account --key-file ${GOOGLE_APPLICATION_CREDENTIALS}
+
+          ./tools/cleanupPolicies.sh ${GOOGLE_CLOUD_DATA_PROJECT}
+      - name: "Import Vault dev secrets"
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.ROLE_ID }}
+          secretId: ${{ secrets.SECRET_ID }}
+          secrets: |
+            secret/dsde/datarepo/dev/sa-key-b64 sa | B64_APPLICATION_CREDENTIALS ;
+      - name: "Add dev credentials to run as Harry Potter test users"
```

Comment:
Nit: Could you change "dev" to "jade-k8-sa" so we know which SA this refers to?

Reply:
Done. Thank you!